music_album.rb

```diff
@@ -0,0 +1,18 @@
+require_relative './item'
+
+class MusicAlbum < Item
+  attr_accessor :on_spotify
+
+  def initialize(name, artist, genre, publish_date, on_spotify)
+    super(publish_date)
+    @name = name
+    @artist = artist
+    @genre = genre
+    @publish_date = publish_date
```

Comment:
In this `initialize` method, I think you don't need to redefine `genre` and `publish_date` since they have been defined in `Item` class.

Reply:
I see more sense in what you are saying. Thanks for the observation.